uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/src/uwrt_mars_rover_drivetrain_hw_real.cpp

```diff
@@ -71,9 +82,16 @@ void UWRTMarsRoverDrivetrainHWReal::write(const ros::Time & /*time*/, const ros:
 
       case UWRTRoverHWDrivetrain::DrivetrainActuatorJointCommand::Type::VELOCITY:
         joint_to_actuator_velocity_interface_.propagate();
-        successful_joint_write = motor_controller_->setVelocity(
-            static_cast<int32_t>(actuator_joint_commands_[joint_name].actuator_data * RADIANS_PER_SECOND_TO_RPM_FACTOR),
-            roboteq_actuator_index_[joint_name]);
+        try {
```

Comment:
Unrelated to this PR but should I also use exceptions in the science ros control real time cpp code when I am sending can messages?

Reply:
Nah dont. Leave a todo with the issue number that implements ros diagnostics